test/extended/util/framework.go

```diff
@@ -274,6 +274,16 @@ func WaitForADeployment(client kclient.ReplicationControllerInterface, name stri
 	}
 }
 
+// WaitForADeploymentToComplete waits for a deployment to complete.
+func WaitForADeploymentToComplete(oc *CLI, name string) bool {
```

Comment:
Perhaps the first argument should match the signature of `WaitForADeployment`?

A `client kclient.ReplicationControllerInterface` is what is needed here, not a `*CLI`. WDYT?